scripts/var_props.py

```diff
@@ -128,6 +128,44 @@ def find_vertical_dimension(dims):
     # end for
     return (var_vdim, vindex)
 
+########################################################################
+def local_name_to_diag_name(prop_dict, context=None):
+########################################################################
+    """
+    Translate a local_name to its default diagnostic name.
+    Currently, this is just equal to the local name. If no local name
+    exists in the property dictionary, a truncation of the standard
+    name is used. (256 characters = max length of NetCDF variable name)
+    >>> local_name_to_diag_name({'local_name':'foo', 'standard_nam':'cloud_optical_depth'})
```

Comment:
Is the missing letter on purpose?
```suggestion
    >>> local_name_to_diag_name({'local_name':'foo', 'standard_name':'cloud_optical_depth'})
```